Fixed issue with last run when there are no alerts (#17526)
demisto/content: Packs/FireEyeCM/ReleaseNotes/1_1_9.md

#### Integrations
##### FireEye Central Management

> - Fixed an issue in the *fetch-incidents* command where alerts that were created 48 hours or more after the last alert, were not fetched.

Reviewer: ```suggestion
- Fixed an issue in the fetch incidents process where alerts were not fetched if they that were created 48 hours or later than the last alert that was fetched.
```
